client/data/settings/action-types.js

```diff
@@ -2,6 +2,7 @@
 
 export default {
 	SET_SETTINGS: 'SET_SETTINGS',
+	SET_SETTINGS_VALUE: 'SET_SETTINGS_VALUE',
```

Comment:
Trying to create less action types - more complete refactor here: https://github.com/Automattic/woocommerce-payments/pull/2047